crates/bench/benches/special.rs

```diff
@@ -48,6 +51,31 @@ fn custom_module_benchmarks(c: &mut Criterion) {
     }
 }
 
+fn custom_db_benchmarks(c: &mut Criterion) {
+    let runtime = start_runtime();
+
+    let config = Config {
+        storage: Storage::Memory,
```

Comment:
Might be worth running these on disk?

You could also split them out:
```rust
for storage in [Storage::Memory, Storage::OnDisk] {
   let config = /* ... */
   // ...
   let storage_label = match storage {
     Storage::Memory => "in_memory",
     Storage::Disk => "on_disk"
   };
   group.bench_function(&format!("db_game/{storage_label}/load={n}"), |b| { /* ... */ });
}
```

the nice thing about this is that you can still ignore some of these settings when running criterion:
`cargo bench --bench special -- 'db_game/in_memory/*'` will only run the in-memory ones.

Reply:
On the other hand, this is gonna be pretty slow -- 8 minutes for (load=10, load=100) x (on_disk, in_memory) -- so it may be worth turning the measurement time down...

Reply:
I fix them now so the test finishes earlier...